hack/dockerized

```diff
@@ -22,7 +22,7 @@ if [ -z ${KUBEVIRT_CRI} ]; then
     fi
 fi
 
-KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2107121244-a7ea2e372"
+KUBEVIRT_BUILDER_IMAGE="quay.io/andreabolognani/kubevirt-builder:2107271047-5ea5d8276"
```

Comment:
You definitely don't want this commit to end up being merged! I think you can safely drop all of my commits that have "FIXME" in the message: the bazeldnf one is later elided when you bump bazeldnf further, and switching to CentOS Stream 8 for the builder image should no longer be necessary thanks to the new sandbox. Better yet, you can push a new builder image generated from your applied series, push it under the official KubeVirt quay.io account and update the script accordingly...

Reply:
Updated to a latest rebuild.